tests/system/small/test_series.py

```diff
@@ -3416,3 +3416,99 @@ def foo(x: int, y: int, df):
     )
 
     assert_series_equal(bf_result, pd_result)
+
+
+@pytest.mark.parametrize(
+    ("data"),
+    [
+        pytest.param([[1, 2, 3], [], numpy.nan, [3, 4]], id="int"),
+        pytest.param(
+            [["A", "AA", "AAA"], ["BB", "B"], numpy.nan, [], ["C"]], id="string"
+        ),
+        pytest.param(
+            [
+                {"A": {"x": 1.0}, "B": "b"},
+                {"A": {"y": 2.0}, "B": "bb"},
+                {"A": {"z": 4.0}},
+                {},
+                numpy.nan,
+            ],
+            id="struct",
+        ),
+    ],
+)
+def test_series_explode(data):
```

Comment:
I'm not sure if any of these tests use the unordered path. Maybe use `to_pandas(ordered=False)` for a test or two? Also for unordered test cases make sure not to ignore index as resetting the index will invoke the ordered path.

Reply:
It looks dtype validation will trigger the unordered path. Also I added the aggregation and `to_pandas(ordered=False)` as you suggested. Thanks!